table_tagger.py

```diff
@@ -88,6 +88,9 @@ def main():
 
     parser.add_argument(
         '--tag-name', required=False, type=str, default='table_name',  help='tag table with tag TAG_NAME (default is "table_name")')
+    
+    parser.add_argument(
+        '--profile', required=False, type=str, default='default', help='set a custom profile name to perform the operation under')
```

Comment:
The default value should be none, what if you want to use the environment variables instead of the client?

Reply:
Updated to `None`